cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -80,6 +80,12 @@ spec:
     # deployed) to set rulesNamespace for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.
     rulesNamespace: rook-ceph
   network:
+    # Whether to encrypt the data in transit across the wire to prevent eavesdropping the data on the network.
```

Comment:
How about something like this for more granularity?

```yaml
network:
  connections:
    encrypted:
      cluster: true
      client: false
      service: false
    compressed:
      enabled: true
      < later we can add things like "algorithm" or whatever and we are not blocked by a single bool >
```

Reply:
I still don't see necessary for separate cluster/client/service settings. Either they should want connections encrypted, or not, but it's flexible now in case we change it later too.
```yaml
network:
  connections:
    encryption:
      enabled: true
    compression:
      enabled: true
```